test/query.test.js

```diff
@@ -2708,17 +2708,144 @@ describe('query', () => {
     }
   })
 
-  // TODO Create tests once work is done in Core
-  test.skip('current_identity', () => {})
+  // API v4 AccessProvider functions
 
-  // TODO Add test once Core work has been done
-  test.skip('has_current_identity', () => {})
+  test('current_identity returns object when authed', async () => {
```

Comment:
@nathaniel-may @marrony Are there more cases we need to test for this function?

Reply:
in theory you should test with JWT tokens too

Reply:
Good idea 👍 cc: @danieltodonnell

Reply:
I've added a `TODO` for this and we can add these tests once we've set up our Auth0/Okta accounts.